### PR TITLE
Docker Image Ref

### PR DESCRIPTION
**Pull request by cdoyle-temple**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -35,11 +35,9 @@ lint:
 build:
   stage: build
   extends: .build_image
-  image: $HARBOR/gitlab-ci/docker:20
   variables:
     DF: ".docker/app/Dockerfile --no-cache"
   services:
```
Comment (sensei100) [inline .gitlab-ci.yml]: @cdoyle-temple I think you should remove this whole services block.  It is already in the gitlab-template provided for us:

Comment (cdoyle-temple) [reply .gitlab-ci.yml]: @sensei100 done

Comment (sensei100) [reply .gitlab-ci.yml]: Sorry, this is removing too much just the services block.  Lines 41-43. And you should still keep the removal of the image on line 38.

Comment (cdoyle-temple) [reply .gitlab-ci.yml]: @sensei100 -- got it

**Review by sensei100 — APPROVED** [review]

**Pull request merged**